packages/xo-server/src/xo-mixins/metadata-backups.js

```diff
@@ -403,13 +129,15 @@ export default class metadataBackup {
 
     let { retentionXoMetadata, retentionPoolMetadata } = job.settings[schedule.id] || {}
 
+    const metadataDefaultSettings = this._backupOptions.metadata.defaultSettings
+
     // it also replaces null retentions introduced by the commit
     // https://github.com/vatesfr/xen-orchestra/commit/fea5117ed83b58d3a57715b32d63d46e3004a094#diff-c02703199db2a4c217943cf8e02b91deR40
     if (retentionXoMetadata == null) {
-      retentionXoMetadata = DEFAULT_RETENTION
+      retentionXoMetadata = metadataDefaultSettings.retentionXoMetadata
     }
     if (retentionPoolMetadata == null) {
-      retentionPoolMetadata = DEFAULT_RETENTION
+      retentionPoolMetadata = metadataDefaultSettings.retentionPoolMetadata
```

Comment:
I don't understand, isn't handled by the new backup code?

Reply:
I should add them to the job. 

The way that we merge the default settings isn't good because if the property is defined and it has `undefined` or `null` as value, this will override the default value.